config.default.yaml

```diff
@@ -24,11 +24,16 @@ scenario:
   # B for biomass supply, I for industry, shipping and aviation
   # solarx or onwindx changes the available installable potential by factor x
   # dist{n} includes distribution grids with investment cost of n times cost in data/costs.csv
+  # solar+c0.5 reduces the capital cost of solar to 50\% of reference value
+  # for myopic/perfect foresight cb states the carbon budget in GtCO2 (cumulative 
```

Comment:
Maybe include example at beginning of line, e.g. cb40ex0?

Reply:
Yes, https://github.com/PyPSA/pypsa-eur-sec/pull/89/commits/faebbc493b78b6ec386e19bae2b1b43ef83e0c2a

Reply:
Great, thanks!